docs/checks/reference/git.md

```diff
@@ -0,0 +1,18 @@
+Git
```

Comment:
Can you add a short blurb at the bottom of this table that describes the default nature of the check? For example, if a user were to configure only the required values, it seems the check will connect to the Git server, clone the specified branch, then pass the check. This will be useful for people reading the docs and knowing the true "default" nature of a check without reading through code.

I recognize that other checks don't do this in the docs but it is something that I always find helpful :)

Reply:
Added: https://github.com/scorestack/scorestack/pull/346/commits/17cb41b03c0280a92fa86eb8632a56c3fb0ff181